@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageTSDHandleControl.sqf

```diff
@@ -30,11 +30,10 @@ switch (_state get "subPageVarPage" select 0) do {
             case "b6": {
                 _state set ["subPageVarPage", TSD_THRT]; //THRT subpage
             };
-            case "r2": {
-                [_heli] call fza_mpd_fnc_handleZoom;
-            };
             case "r3": {
-                _persistState set ["ctr", 1 - (_persistState get "ctr")];
+                if (_state get "ShowScale" == 0) then {
```

Comment:
This doesn't seem to work for me. If I, on the WPT subpage, go through and press the R2 for me to open the scale and then R3, the CTR is changed too